### PR TITLE
Fix sync_with_audio spelling and grammar

### PR DESCRIPTION
**Pull request by dominiccooney**
- Use correct spelling and grammar
- Omit meaningless intensifier "very"
- Refer consistently to chunks; don't casually call them blocks
- Some minor changes for flow

### CODE REVIEW

File: tutorials/audio/sync_with_audio.rst [hunk 1]
```diff
@@ -6,28 +6,27 @@ Sync the gameplay with audio and music
...
+In any application or game, sound and music playback will have a slight delay. For games, this delay is often so small that it is negligible. Sound effects will come out a few milliseconds after any play() function is called. For music this does not matter as in most games it does not interact with the gameplay.
 
-Still, for some games (mainly, rhythm games), it may be required to synchronize player actions with something happening in a song (usually in sync with the BPM). For this, having more precise timing information for an exact playback position is very welcome.
-
-Achieving very low playback timing precision is very difficult. This because many factors are at play during audio playback:
+Still, for some games (mainly, rhythm games), it may be required to synchronize player actions with something happening in a song (usually in sync with the BPM). For this, having more precise timing information for an exact playback position is useful.
 
+Achieving very low playback timing precision is difficult. This because many factors are at play during audio playback:
```
Comment (aaronfranke) [inline tutorials/audio/sync_with_audio.rst]: Pre-existing text, but should be "This **is** because"

File: tutorials/audio/sync_with_audio.rst [hunk 2]
```diff
@@ -6,28 +6,27 @@ Sync the gameplay with audio and music
...
 
 * Audio is mixed in chunks (not continuously), depending on the size of audio buffers used (check latency in project settings).
-* Audio has latency after mixed, it doesn't immediately play after mixed.
+* Audio does not play a mixed chunk immediately.
```
Comment (aaronfranke) [inline tutorials/audio/sync_with_audio.rst]: "Audio" is the object, not the subject. Perhaps "Mixed chunks of audio are not played immediately".

**Comment by dominiccooney** [comment]
Thanks for the review! I have incorporated your comments. I squashed and rebased, if godot prefers a different workflow please let me know.

**Comment by NathanLovato** [comment]
Thank you 🙂

**Pull request merged**